sdk/storage/azure-storage-file-share/tests/test_nfs_async.py

```diff
@@ -0,0 +1,262 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+from azure.core.exceptions import ClientAuthenticationError, HttpResponseError, ResourceExistsError, ResourceNotFoundError
+from azure.storage.fileshare import ContentSettings, ShareServiceClient
+from azure.storage.fileshare.aio import ShareServiceClient as AsyncShareServiceClient
+from azure.storage.fileshare.aio import ShareFileClient, ShareDirectoryClient
+
+from devtools_testutils.aio import recorded_by_proxy_async
+from devtools_testutils.storage.aio import AsyncStorageRecordedTestCase
+from settings.testcase import FileSharePreparer
+
+
+TEST_INTENT = 'backup'
+TEST_FILE_PREFIX = 'file'
+TEST_DIRECTORY_PREFIX = 'directory'
+
+
+class TestStorageFileNFSAsync(AsyncStorageRecordedTestCase):
+
+    fsc: ShareServiceClient = None
```

Comment:
This should be typed `AsyncShareServiceClient`

Reply:
Good catch, thank you!